src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2100,6 +2100,32 @@ theorem toNat_abs {x : BitVec w} : x.abs.toNat = if x.msb then 2^w - x.toNat els
     rw [Nat.mod_eq_of_lt (by omega)]
   · simp [h]
 
+theorem getLsbD_abs {i : Nat} {x : BitVec w} :
```

Comment:
I wonder whether we want to have the `getLsbD`/`msb` etc. inside or outside of the if here. If (haha) it was inside we could run `simp` after applying this theorem to reason about both branches of the if quite easily. If one of them happens to simplify to true or false more theorems on if would fire as well that make it disappear entirely.